frontend/viewer/src/project/browse/SenseRow.svelte

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+  import Badge from '$lib/components/ui/badge/badge.svelte';
+  import type { ISense } from '$lib/dotnet-types';
+  import {useWritingSystemService} from '$lib/writing-system-service.svelte';
+  import type {Snippet} from 'svelte';
+  import ListItem, {type ListItemProps} from '$lib/components/ListItem.svelte';
+
+  interface Props extends ListItemProps {
+    sense: ISense;
+    badge?: Snippet,
+  };
+
+  const {
+    sense,
+    badge,
+    ...rest
```

Comment:
is it intentional that you leave `children` in here therefore if it's always ignored (had to test that). Meaning your use in `EntryOrSensePicker` doesn't make since as this `<SenseRow>test</SenseRow>` just throws away the inner html.